test/addon/ckeztest/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "devDependencies": {
-    "testcafe": "~1.13.0"
+    "testcafe": "~15.1.317922"
```

Comment:
This is wrong. As of Jan 12, 2023, testcafe is at 2.2.0. 15.1.317922 points to https://www.npmjs.com/package/testcafe/v/15.1.317922. Which has been deprecated. The official package should be https://www.npmjs.com/package/testcafe.

Reply:
Since zkckeditor 4.21.0.0, testcafe is bumped to "~2.2.0".